lib/resolvers/auth_mutations/saveFcmToken.js

```diff
@@ -0,0 +1,20 @@
+const User = require('../../models/User');
+const { NotFoundError } = require('errors');
+const requestContext = require('talawa-request-context');
+
+module.exports = async (parent, args, context) => {
+  const user = await User.findById(context.userId);
+  if (!user) {
+    throw new NotFoundError(
+      requestContext.translate('user.notFound'),
+      'user.notFound',
+      'user'
```

Comment:
hey @guptapriyanshu7, swap these strings with the ones from the constants file ~

Reply:
Fixed.